devicetypes/smartthings/hue-bridge.src/hue-bridge.groovy

```diff
@@ -15,19 +15,27 @@ metadata {
 		// TODO: define status and reply messages here
 	}
 
-	tiles {
+	tiles(scale: 2) {
+     	multiAttributeTile(name:"rich-control"){
+			tileAttribute ("", key: "PRIMARY_CONTROL") {
```

Comment:
Have you tested this without a name here? I'm afraid it will break. It would be good testing on older clients too.